.gitignore

```diff
@@ -15,3 +15,7 @@ build/
 *.egg-info/
 **/dist/
 __pycache__
+
+# Ignore project files
```

Comment:
This does not belong into a project-wide gitignore file as it is specific to each developer. You can define a global `core.excludesfile` matching your chosen tool set.

Reply:
I've temporarily added it when starting to work on a branch - thanks, will remove.